### PR TITLE
LOC CHECKIN | dotnet/project-system  master | 20190606

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.pl.xlf [hunk 1]
```diff
@@ -79,12 +79,12 @@
...
       <trans-unit id="26">
         <source>C# Project Files (*.csproj);*.csproj</source>
-        <target state="translated">Pliki projektu C# (*.csproj); *.csproj</target>
+        <target state="new">C# Project Files (*.csproj);*.csproj</target>
         <note />
       </trans-unit>
       <trans-unit id="27">
         <source>.NET Core 2015 Project Files (*.xproj);*.xproj</source>
-        <target state="translated">Pliki projektu .NET Core 2015 (*.xproj); *.xproj</target>
+        <target state="new">.NET Core 2015 Project Files (*.xproj);*.xproj</target>
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.pl.xlf]: This, and line 82, look like a regression in translation. Is this deliberate?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf [hunk 2]
```diff
@@ -69,7 +69,7 @@
       </trans-unit>
       <trans-unit id="24">
         <source>F# Project Files (*.fsproj);*.fsproj</source>
-        <target state="translated">F# 專案檔 (*.fsproj); *.fsproj</target>
+        <target state="new">F# Project Files (*.fsproj);*.fsproj</target>
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: Another potential regression.

Comment (janbaltus) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: Thank you for spotting this.
I logged ticket as this seems as tool issue.

Comment (janbaltus) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: We updated PR not to block merge

**Pull request merged**